### DISCUSSION TITLE
Copilot free access is not renewed

### DISCUSSION THREAD

**Discussion by r0x0r** [community/community#116622] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I am an author of a popular open source projec. Last year I was awarded free Copilot access, but now it has not been renewed. The Copilot FAQ states that _Once awarded, if you are still a maintainer of a popular open source project when your initial 12 months subscription expires then you will be able to renew your subscription for free._ I still maintain the project, but I cannot find an option for renewing subscription for free. Could you check if I still qualify for free Copilot?

**Comment by Akash1134** [comment, marked as answer]
Hi @r0x0r ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
